projects/GKCore/GKCore/Controllers/AssociationEditDlgController.cs

```diff
@@ -63,8 +62,7 @@ public override bool Accept()
                 }
 
                 fAssociation.Relation = rel;
-                fAssociation.Individual = fTempPerson;
-                //fBase.Context.Tree.SetPtrValue(fAssociation, fTempPerson);
+                fBase.Context.Tree.SetPtrValue(fAssociation, fTempPerson);

```

Comment:
I'm not sure why you have not replaced this by yourself

Reply:
Bug with using NSubstitute in the /projects/GKTests/GKCore/Controllers/AssociationEditControllerTests.cs/Test_AssociationEditDlgController()